content/source/docs/enterprise/saml/identity-provider-configuration-adfs.html.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: enterprise2
+page_title: "ADFS Configuration - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-private-saml-identity-provider-configuration-adfs"
+---
+
+# ADFS Configuration
+
+This guide explains how to configure Active Directory Federated Services (ADFS) in order to use it as an Identity Provider (IdP) for PTFE's SAML authentication feature. The screenshots below were taken on Windows Server 2016, and the UI may not look the same on previous Windows versions.
+
+This document assumes that you have already installed and configured ADFS and that you are using PTFE version v201807-1 or later.
+
+## Gather ADFS information
+
+1. On the ADFS server, start the Server Manager.
+  ![saml_0](./images/saml_0.png)
+2. Click "Tools" -> "AD FS Management"
+  .![saml_1](./images/saml_1.png)
+3. Expand the `Service` object and click "Endpoints".
+  ![saml_2](./images/saml_2.png)
+4. Make a note of the `URL Path` for Type `SAML 2.0/WS-Federation`. (If you are using the default settings, this will be `/adfs/ls/`.)
+5. Switch from "Endpoints" to "Certificates" and choose the one under `Token-signing`.
+  ![saml_3](./images/saml_3.png)
+6. Right click "View Certificate".
+  ![saml_5](./images/saml_5.png)
+7. In the Certificate dialog, select the Details tab and click "Copy to File".
+  ![saml_6](./images/saml_6.png)
+8. In the Certificate Export Wizard, click "Next", select "Base-64 encoded X.509 (.CER)" and click "Next" again.
+  ![saml_8](./images/saml_8.png)
+9. Pick a location to save the file and click "Next".
+  ![saml_9](./images/saml_9.png)
+10. Review the settings and click "Finish".
+  ![saml_10](./images/saml_10.png)
+
+## Configure PTFE
+
+1. Visit `https://<TFE HOSTNAME>/app/admin/saml`.
+2. Set "Single Sign-on URL" to `https://<ADFS hostname>/<URL Path>`.
+3. Set "Single Log-out URL" to `https://<ADFS hostname>/<URL Path>?wa=wsignout1.0`.
+4. Paste the contents of the saved certificate in "IDP Certificate".
+5. Scroll to the bottom of the screen and click "Save SAML Settings".
+
+## Configure ADFS
```

Comment:
I wonder if we want to move this up above Configure PTFE, the way Zacc has it arranged in the OneLogin doc? That would save some of the repeats, and maybe be more efficient.

Reply:
PTFE needs to be configured before we point ADFS to the metadata URL, otherwise it will not work.

Reply:
Oh, got it! I totally missed the metadata URL use, very clever. This makes sense, then.